docs/user/gcp/install_upi.md

```diff
@@ -0,0 +1,506 @@
+if [ -f transaction.yaml ]; then rm transaction.yaml; fi
+gcloud dns record-sets transaction start --zone ${INFRA_ID}-int-managed-zone
+gcloud dns record-sets transaction add "${MASTER0_IP}" --name etcd-0.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-int-managed-zone
+gcloud dns record-sets transaction add "${MASTER1_IP}" --name etcd-1.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-int-managed-zone
+gcloud dns record-sets transaction add "${MASTER2_IP}" --name etcd-2.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-int-managed-zone
+gcloud dns record-sets transaction add \
+  "0 10 2380 etcd-0.${CLUSTER_NAME}.${BASE_DOMAIN}." \
+  "0 10 2380 etcd-1.${CLUSTER_NAME}.${BASE_DOMAIN}." \
+  "0 10 2380 etcd-2.${CLUSTER_NAME}.${BASE_DOMAIN}." \
+  --name _etcd-server-ssl._tcp.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type SRV --zone ${INFRA_ID}-int-managed-zone
+gcloud dns record-sets transaction execute --zone ${INFRA_ID}-int-managed-zone
+```
+
+## Monitor for `bootstrap-complete` and Initialization
+
+```console
+$ openshift-install wait-for bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+```
+## Pivot load balancers to masters
+
+At this point, only the bootstrap node is in the load balancers. This was done
+so the masters could boot ignition from the bootstrap instance. Now we can
+pivot the load balancers to the masters.
+
+```console
+# Add masters
+gcloud compute target-pools add-instances ${INFRA_ID}-api-target-pool --instances=${INFRA_ID}-master-0
+gcloud compute target-pools add-instances ${INFRA_ID}-ign-target-pool --instances=${INFRA_ID}-master-0
+gcloud compute target-pools add-instances ${INFRA_ID}-api-target-pool --instances=${INFRA_ID}-master-1
+gcloud compute target-pools add-instances ${INFRA_ID}-ign-target-pool --instances=${INFRA_ID}-master-1
+gcloud compute target-pools add-instances ${INFRA_ID}-api-target-pool --instances=${INFRA_ID}-master-2
+gcloud compute target-pools add-instances ${INFRA_ID}-ign-target-pool --instances=${INFRA_ID}-master-2
+
+# Remove bootstrap instance
+gcloud compute target-pools remove-instances ${INFRA_ID}-api-target-pool --instances=${INFRA_ID}-bootstrap
+gcloud compute target-pools remove-instances ${INFRA_ID}-ign-target-pool --instances=${INFRA_ID}-bootstrap
+```
+## Destroy Bootstrap Resources
+
+At this point, you should delete the bootstrap resources.
+
+```console
+gcloud deployment-manager deployments delete ${INFRA_ID}-bootstrap
+gsutil rm gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign
+gsutil rb gs://${INFRA_ID}-bootstrap-ignition
+```
+
+## Launch Additional Compute Nodes
+
+You may create compute nodes by launching individual EC2 instances discretely
```

Comment:
this looks like a copy/paste error from aws (ie ec2)

Reply:
Good catch, thanks.